whirlpool/washerdryer.py

```diff
@@ -79,6 +81,59 @@ class MachineState(Enum):
     MachineState.SystemInit: ATTRVAL_MACHINE_STATE_SYSTEMINIT,
 }
 
+MACHINE_STATE_MAP_PRETTY = {
```

Comment:
The lib should not have "pretty" strings, but only Enums, because it is just a data provider and pretty strings are not the best way to pass data between code components. HA should take care of prettifying, since HA is the one showing the UI to the user.
If you don't mind, I ask you to please do remove the "pretty" things :sweat_smile: